.travis.yml

```diff
@@ -1,17 +1,27 @@
+dist: xenial
 language: python
 git:
   depth: 50 # Need old commits for determining version of untagged builds
 env: DJANGO=1.11.9
 matrix:
   include:
   - python: 2.7
-  - python: 3.4
   - python: 2.7 # Old Middleware classes
     env: DJANGO=1.9
-  - python: 3.4 # Django 2 support
-    env: DJANGO==2.0
+  - python: 3.5 
+    env: DJANGO=2.0
+  - python: 3.6
+    env: DJANGO=2.0
+  - python: 3.7
+    env: DJANGO=2.0
```

Comment:
Hi @asherf,

I just checked [supported versions of Django](https://www.djangoproject.com/download/#supported-versions) and found that 2.0 is not supported since `April 1, 2019`.
Meanwhile, Django 1.11 is an LTS release and it will be supported until `April 2020`, so it would be nice to have tests for 1.11 too.

Reply:
updated